Assets/MRTK/Providers/Oculus/XRSDK/MRTK-Quest/Editor/OculusXRSDKHandtrackingConfigurationChecker.cs

```diff
@@ -155,24 +209,33 @@ internal static void ConfigureOculusIntegration()
         }
 
         /// <summary>
-        /// Updates the assembly definitions to mark the Oculus Integration Asset as present or not present
+        /// Checks if the Oculus Integration Asset as present or not present
         /// </summary>
         /// <returns>true if Assets/Oculus/OculusProjectConfig exists, false otherwise</returns>
-        internal static bool ReconcileOculusIntegrationDefine()
+        internal static bool DetectOculusIntegrationDefine()
         {
             FileInfo[] files = FileUtilities.FindFilesInAssets(OculusIntegrationProjectConfig);
 
-            if (files.Length > 0)
+            return files.Length > 0;
+        }
+
+        /// <summary>
+        /// Updates the assembly definitions to mark the Oculus Integration Asset as present or not present
+        /// </summary>
+        /// <returns>true if Assets/Oculus/OculusProjectConfig exists, false otherwise</returns>
+        internal static void ReconcileOculusIntegrationDefine()
+        {
+            bool isOculusPresent = DetectOculusIntegrationDefine();
+
+            if (isOculusPresent)
```

Comment:
```suggestion
            if (DetectOculusIntegrationDefine())
```

Reply:
But this change might not be necessary based on https://github.com/microsoft/MixedRealityToolkit-Unity/pull/8616/files#r493043435